wcivf/apps/elections/import_helpers.py

```diff
@@ -0,0 +1,250 @@
+import sys
+
+from django.conf import settings
+from django.db import transaction
+
+from elections.helpers import JsonPaginator, EEHelper
+from elections.models import PostElection, Election, Post, VotingSystem
+from people.models import Person, PersonPost
+
+
+class YNRElectionImporter:
+    """
+    Takes a JSON object from YNR and creates or updates an election object
+    from it.
+
+    Manages caching, and updating metadata from EE
+
+    """
+
+    def __init__(self, ee_helper=None):
+        if not ee_helper:
+            ee_helper = EEHelper()
+        self.ee_helper = ee_helper
+        self.election_cache = {}
```

Comment:
Isn't this just duplicating functionality that is already implemented in `EEHelper`? (same with `post_cache` )

https://github.com/DemocracyClub/WhoCanIVoteFor/blob/ccf440c333f47fc646990e3a757e1e8493cadc71/wcivf/apps/elections/helpers.py#L15-L16

Reply:
No, because one is the `election` from YNR and the other is the `election` from EE…